src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Roslyn.Test.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Emit
+{
+    public class EndToEndTests: EmitMetadataTestBase
+    {
+        [WorkItem(16669, "https://github.com/dotnet/roslyn/issues/16669")]
+        [Fact]
+        public void OverflowOnFluentCall()
+        {
+#if DEBUG
+            int numberFluentCalls = 290;
+#else
+            int numberFluentCalls = 590;
```

Comment:
@jaredpar We figured out the discrepancy (in my setup, the test itself was built in Debug, while the libraries it used were Retail). With the same setup, Ty gets the same number.